packages/medusa/src/api/routes/store/orders/create-order.js

```diff
@@ -0,0 +1,46 @@
+import { MedusaError, Validator } from "medusa-core-utils"
+
+export default async (req, res) => {
+  const schema = Validator.object().keys({
+    status: Validator.string().optional(),
+    email: Validator.string()
+      .email()
+      .required(),
+    billing_address: Validator.address().required(),
+    shipping_address: Validator.address().required(),
+    items: Validator.array().required(),
+    region: Validator.string().required(),
+    discounts: Validator.array().optional(),
+    customer_id: Validator.string().required(),
+    payment_method: Validator.object()
+      .keys({
+        provider_id: Validator.string().required(),
+        data: Validator.object().optional(),
+      })
+      .required(),
+    shipping_method: Validator.array()
+      .items({
+        provider_id: Validator.string().required(),
+        profile_id: Validator.string().required(),
+        price: Validator.number().required(),
+        data: Validator.object().optional(),
+        items: Validator.array().optional(),
+      })
+      .required(),
+    metadata: Validator.object().optional(),
+  })
```

Comment:
I think we should convert carts to orders. So instead of having to gather all this info in the client, we can just pass the `cartId`, retrieve the cart and create the order